telegram/_webappdata.py

```diff
@@ -27,27 +27,26 @@ class WebAppData(TelegramObject):
     """Contains data sent from a `Web App <https://core.telegram.org/bots/webapps>`_ to the bot.
 
     Objects of this class are comparable in terms of equality. Two objects of this class are
-    considered equal, if their :attr:`data` and :attr:`button_text` are equal.
+    considered equal, if their :attr:`data` and :attr:`button_text` attributes are equal.
 
     .. seealso:: :any:`Webappbot Example <examples.webappbot>`
 
     .. versionadded:: 20.0
 
     Args:
-        data (:obj:`str`): The data. Be aware that a bad client can send arbitrary data in this
-            field.
+        data (:obj:`str`): The data. *Be aware that a bad client can send arbitrary data in this
+            field.*
         button_text (:obj:`str`): Text of the :paramref:`~telegram.KeyboardButton.web_app` keyboard
             button, from which the Web App was opened.
 
     Attributes:
-        data (:obj:`str`): The data. Be aware that a bad client can send arbitrary data in this
-            field.
+        data (:obj:`str`): The data. *Be aware that a bad client can send arbitrary data in this
+            field.*
```

Comment:
The official docs don't do any formatting on this so I don't think we should either

Reply:
What do you mean by _official docs_? I looked at my local sphinx build and that had the formatting in there.

Reply:
[Here](https://core.telegram.org/bots/api#webappdata). We usually copy the same formatting used in our docs.

Reply:
Thanks. The missing link... Missing from https://docs.python-telegram-bot.org/en/v20.0a1/contributing.html#documenting and https://docs.python-telegram-bot.org/en/v20.0a1/contributing.html. The documentation I consulted setting out. No problem. Will revert. Although, I think it wouldn't hurt having the formatting in attributes (sphinx) only.